src/java/org/apache/cassandra/service/NativeTransportService.java

```diff
@@ -149,7 +149,8 @@ public void destroy()
         servers = Collections.emptyList();
 
         // shutdown executors used by netty for native transport server
-        workerGroup.shutdownGracefully(3, 5, TimeUnit.SECONDS).awaitUninterruptibly();
+        if (workerGroup != null)
```

Comment:
this was done for tests as when we startup with an issue, the test shutdown fails as this is null.